music21/musicxml/m21ToXml.py

```diff
@@ -1422,6 +1426,10 @@ def parse(self):
         self.scorePreliminaries()
 
         if s.hasPartLikeStreams():
+            # Prepopulate partExporterList so that joinable groups can be identified
+            # before attempting to identify and count instruments
+            self._populatePartExporterList()
+            self.joinedGroups = self.joinableGroups()
```

Comment:
should the name `joinedGroups` be changed since they're not actually joined yet.  In case they're queried.

Reply:
good idea!